prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+Emails
+Estimate: 20 minutes
+Actual:   44 minutes
+"""
+
+
+def main():
+    "Create dictionary of emails_to_names"
+    email_to_names = {}
+    email = input("Email: ")
+    while email != "":
+        name = extract_name_from_email(email)
+        confirm_name = input(f"Is your name {name}? (Y/N)")
```

Comment:
Should be (Y/n), as we want Y to be the default value (note the case difference).  The Y being capitalized means it is the default value, and so we will treat a response of nothing, or "" as written in code, as a Yes.